files/hello_world_example_app/README.md

```diff
@@ -0,0 +1,36 @@
+This is a hello ROS example app. It subscribes to `/chatter` topic and when a message is received, the following message is published in `/a_chatter`:
+
+    data: "hello world from android ndk __COUNTER_VALUE__"
+
+USAGE
+-------
+
+1. IP addresses are harcoded, so you must edit the master URI and the cellphone ip address in the following file:
```

Comment:
It's not necessarily a cell phone. Suggestion: "Android device" or simply "device"